.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -232,14 +230,14 @@ jobs:
           aws configure set output json
           aws configure set aws_access_key_id $CLOUDFLARE_R2_ACCESS_KEY_ID
           aws configure set aws_secret_access_key $CLOUDFLARE_R2_SECRET_ACCESS_KEY
-      - name: Deploy HTML Report
-        run: aws s3 sync ./playwright-report s3://${{ vars.CLOUDFLARE_R2_BUCKET_NAME }}/${{ github.head_ref }}/visual-test-report --endpoint-url ${{ vars.CLOUDFLARE_R2_ENDPOINT }}
+      - name: Deploy Report
+        run: aws s3 sync ./playwright-report s3://${{ vars.CLOUDFLARE_R2_BUCKET_NAME }}/${{ github.head_ref }}/visual-report --endpoint-url ${{ vars.CLOUDFLARE_R2_ENDPOINT }}
       - uses: marocchino/sticky-pull-request-comment@52423e01640425a022ef5fd42c6fb5f633a02728
         with:
-          header: visual-test-report
+          header: visual-report
           message: |+
-            ## ${{ needs.test-visuals.outputs.failed && '⚠️' || '✅' }} Visual Test Report
-            https://${{ vars.CLOUDFLARE_CUSTOM_DOMAIN }}/${{ github.head_ref }}/visual-test-report
+            ## ${{ needs.test-visuals.outputs.failed && '⚠️' || '✅' }} Visual Report
```

Comment:
Removed "Test" from the name because we'll soon have three reports:

<img width="921" height="645" alt="image" src="https://github.com/user-attachments/assets/41e153c6-48e7-4adc-87fc-463e6e9d99eb" />